lib/Alchemy/Phrasea/Collection/CollectionService.php

```diff
@@ -268,27 +268,28 @@ public function unmountCollection(CollectionReference $reference)
      */
     public function grantAdminRights(CollectionReference $reference, User $user)
     {
-        $rights = [
-            "canputinalbum"   => "1",
-            "candwnldhd"      => "1",
-            "nowatermark"     => "1",
-            "candwnldpreview" => "1",
-            "cancmd"          => "1",
-            "canadmin"        => "1",
-            "actif"           => "1",
-            "canreport"       => "1",
-            "canpush"         => "1",
-            "basusr_infousr"  => "",
-            "canaddrecord"    => "1",
-            "canmodifrecord"  => "1",
-            "candeleterecord" => "1",
-            "chgstatus"       => "1",
-            "imgtools"        => "1",
-            "manage"          => "1",
-            "modify_struct"   => "1"
-        ];
-
-        $this->app->getAclForUser($user)->update_rights_to_base($reference->getBaseId(), $rights);
+        $this->app->getAclForUser($user)->update_rights_to_base(
+            $reference->getBaseId(),
+            [
+                "basusr_infousr"          => "",    // todo : wtf
+                \ACL::CANPUTINALBUM       => true,
```

Comment:
It would be better to use _ between words in constants, ie \ACL::CAN_PUT_IN_ALBUM, for readability